pytorch_lightning/profiler/pytorch.py

```diff
@@ -450,6 +451,9 @@ def summary(self) -> str:
 
         self._delete_profilers()
 
+        if self._schedule is not None:
+            self._schedule.reset()
+
```

Comment:
Should this be in `_delete_profilers` instead?

Reply:
I hesitated, but yes why not.